apps/client/src/common/utils/eventsManager.ts

```diff
@@ -29,3 +41,33 @@ export const cloneEvent = (event: OntimeEvent, after?: string): ClonedEvent => {
     custom: {},
   };
 };
+
+export const cloneEntry = (entry: OntimeRundownEntry, after?: string): ClonedEvent | ClonedBlock | ClonedDelay => {
+  if (isOntimeEvent(entry)) {
+    return {
+      type: SupportedEvent.Event,
+      title: entry.title,
+      note: entry.note,
+      timeStart: entry.timeStart,
+      duration: entry.duration,
+      timeEnd: entry.timeEnd,
+      timerType: entry.timerType,
+      timeStrategy: entry.timeStrategy,
+      linkStart: entry.linkStart,
+      endAction: entry.endAction,
+      isPublic: entry.isPublic,
+      skip: entry.skip,
+      colour: entry.colour,
+      after,
+      revision: 0,
+      timeWarning: entry.timeWarning,
+      timeDanger: entry.timeDanger,
+      custom: {},
+    };
+  } else if (isOntimeBlock(entry)) {
+    return { type: SupportedEvent.Block, title: entry.title, after };
```

Comment:
I am not sure clone should be a operation for delays and blocks. It offers little benefit

Reply:
yeah but I think it is good to have the shortcuts be consistent

if it doesn't bother you too much I would like to keep it